src/main/resources/org/sonatype/nexus/ci/nxrm/Messages.properties

```diff
@@ -21,3 +21,11 @@ NexusPublisherWorkflowStep.FunctionName=nexusPublisher
 NexusPublisherWorkflowStep.DisplayName=Nexus Repository Manager Publisher
 
 NexusPublisherBuildStep.DisplayName=Nexus Repository Manager Publisher
+
+CreateTag.DisplayName=Create Tag (Nexus Repository Manager 3.x)
+CreateTag.Validation.TagNameEmpty=Tag name cannot be blank
+CreateTag.Validation.TagNameInvalid=Tag name can only contain letters, numbers, underscores, hyphens and dots and \
+  cannot start with an underscore or dot, and must be 256 characters or less
+CreateTag.Validation.TagAttributesJson=Could not parse attributes as JSON
```

Comment:
Is the `tagnameinvalid` entry still needed given the validation is no longer being performed? Maybe I'm overlooking something.